composer.json

```diff
@@ -13,27 +13,25 @@
     }
   ],
   "require": {
-    "drupal/auto_entitylabel": "3.x-dev",
-    "drupal/entity_print": "2.x-dev",
-    "drupal/views_block_filter_block": "^1.0@beta",
+    "drupal/auto_entitylabel": "3.x-dev#1dd28579f4559cc03d09ab20143b1feb48644fac",
+    "drupal/ds": "^3.5",
+    "drupal/entity_print": "^2.1",
+    "drupal/page_manager": "dev-4.x#78b492786c841c5d2eb07f6d977275a76b79e46f",
+    "drupal/views_block_filter_block": "^1.0",
     "drupal/views_taxonomy_term_name_depth": "^6.0",
     "drupal/views_infinite_scroll": "^1.6",
-    "drupal/viewfield": "^3.0@beta",
-    "su-sws/jumpstart_ui": "^8.1.0@alpha"
+    "drupal/viewfield": "^3.0-beta5",
+    "su-sws/jumpstart_ui": "dev-8.x-1.x"
```

Comment:
Using dev version convention here but we will cut a master branch for the release.